<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewShotWindow</class>
 <widget class="QWidget" name="NewShotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Shot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="ProjNameLbl">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Project / </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HBoxName">
     <item>
      <widget class="QLabel" name="ShotNameLbl">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ShotNameEdit">
       <property name="placeholderText">
        <string>new shot name</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="NewShotBtn">
     <property name="text">
      <string>Create New Shot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
